### PR TITLE
improve cleanup docs

### PR DESCRIPTION
**Pull request by tcrayford**
This fixes a disrepancy in the documentation - Vault *does* have dead
server cleanup these days.

### CODE REVIEW

File: website/content/docs/internals/integrated-storage.mdx [hunk 1]
```diff
@@ -137,10 +137,12 @@ forwarded to the leader.
 Similar to other storage backends, data that is written to the Raft log and FSM
 will be encrypted by Vault's barrier.
 
-Vault does not currently offer automated dead server cleanup. If you wish to
-decommission a node, or a node dies and must be replaced, the node must manually
-be removed from the cluster with the `remove peer`
-[command](/docs/commands/operator/raft#remove-peer).
+To cleanup dead servers automatically, enable [dead server cleanup in the
```
Comment (raskchanky) [inline website/content/docs/internals/integrated-storage.mdx:140]:
```suggestion
To clean up dead servers automatically, enable [dead server cleanup in the
```

**Review by yhyakuna — APPROVED** [review]
LGTM

**Comment by aphorise** [comment]
Still applicable please merge.

File: website/content/docs/internals/integrated-storage.mdx [hunk 2]
```diff
@@ -137,10 +137,12 @@ forwarded to the leader.
...
+autopilot configuration](https://learn.hashicorp.com/tutorials/vault/raft-autopilot?in=vault/raft).
```
Comment (tomhjp) [inline website/content/docs/internals/integrated-storage.mdx:141]:
```suggestion
autopilot configuration](/vault/tutorials/raft/raft-autopilot).
```

**Review by tomhjp — APPROVED** [review]

File: website/content/docs/internals/integrated-storage.mdx (hunk 2 above)
Comment (aphorise) [reply website/content/docs/internals/integrated-storage.mdx]: Corrected to use `/vault/tutorials/raft/raft-autopilot` instead - thanks @tomhjp

File: website/content/docs/internals/integrated-storage.mdx [hunk 3]
```diff
@@ -138,9 +138,12 @@ forwarded to the leader.
 Similar to other storage backends, data that is written to the Raft log and FSM
 will be encrypted by Vault's barrier.
 
-Vault does not currently offer automated dead server cleanup. If you wish to
-decommission a node, or a node dies and must be replaced, the node must manually
-be removed from the cluster with the `remove peer` [command](/vault/docs/commands/operator/raft#remove-peer).
+To cleanup dead servers automatically, enable [dead server cleanup in the
```
[the remark this answers is not in the document]
Comment (aphorise) [reply website/content/docs/internals/integrated-storage.mdx]: Thanks for spotting that @tomhjp  - I was more focused on removing the older / inaccurate textual reading as per the original intent of the author of his PR - and didnt do a comparative validation for the same information being already present (which were not there when this PR was originally done).

**Review by tomhjp — APPROVED** [review]
Thanks!

**Pull request merged**